packages/algob/sample-project/ts/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "sample-project-ts",
+  "version": "0.0.1",
+  "main": "index.js",
+  "license": "MIT",
+  "devDependencies": {
+    "mocha": "^8.1.1"
+  },
+  "scripts": {
+    "algob": "algob",
+    "test": "mocha",
```

Comment:
do we need this ?
algob should be able to run these tests.

Reply:
ok let's remove.